packages/fxa-content-server/webpack.config.js

```diff
@@ -9,9 +9,8 @@ const config = require('./server/lib/configuration').getProperties();
 const CopyPlugin = require('copy-webpack-plugin');
 
 const ENV = config.env;
-const WEBPACK_MODE_OVERRIDE = config.webpackModeOverride;
 const webpackConfig = {
-  mode: WEBPACK_MODE_OVERRIDE || ENV,
+  mode: 'development', // TODO: Revert once debugging is done
```

Comment:
Based on docs, I think we want this mode. It will still generated unminimized files.